src/main/typescript/node_modules/@atomist/rug/tree/PathExpression.ts

```diff
@@ -158,12 +158,15 @@ interface DynamicType {
 
 }
 
+import { MicroMatcher } from "./Microgrammars"
 /**
  * Dynamic type for a microgrammar. Pass to PathExpression.addType method
  */
 class Microgrammar implements DynamicType {
 
-    constructor(public name: string, public grammar: string, public submatchers: any = {}) {}
+    constructor(public name: string,
```

Comment:
To use the constructor from TypeScript?
I didn't write this, and I'm being careful not to break the existing